package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "lit": "^2.0.2",
+    "@patternfly/pfe-core": "next",
```

Comment:
now using pfe-core in some of our components.

Reply:
++